assets/src/components/d3/createHistogram.js

```diff
@@ -92,22 +92,15 @@ function createHistogram ({ data, width, height, el, tip, xAxisLabel, yAxisLabel
       .attr('stroke-width', '2')
     svg.append('text')
       .attr('x', x(myGrade) - 110)
-      .attr('d', '1em')
```

Comment:
> I noticed a small bug with the My Grade label overlapping with line, so I just rounded the My Grade to be one decimal place (e.g. 80.88% => 80.9%).

Probably that is happening because i removed `d` attribute . I just wanted the Mygrade text to be close line